src/math/Math.js

```diff
@@ -31,7 +31,7 @@ var _Math = {
 				lut[ d2 & 0x3f | 0x80 ] + lut[ d2 >> 8 & 0xff ] + '-' + lut[ d2 >> 16 & 0xff ] + lut[ d2 >> 24 & 0xff ] +
 				lut[ d3 & 0xff ] + lut[ d3 >> 8 & 0xff ] + lut[ d3 >> 16 & 0xff ] + lut[ d3 >> 24 & 0xff ];
 
-			// .toUpperCase() here flattens concatenated strings to save heap memory space. 
+			// .toUpperCase() here flattens concatenated strings to save heap memory space.
```

Comment:
Maybe you can remove this unrelated change from the PR?

Reply:
@Mugen87 this change allows linting to pass.

Reply:
Oh, then it's of course fine :blush: